File: backend/exence/src/main/java/com/exence/finance/modules/auditlog/dto/AuditLogFilter.java
package com.exence.finance.modules.auditlog.dto;

import com.exence.finance.modules.auditlog.enums.AuditableEntityType;
import com.exence.finance.modules.auditlog.enums.ChangeType;
import java.time.LocalDate;

public record AuditLogFilter(
        AuditableEntityType entityType, LocalDate from, LocalDate to, ChangeType changeType, String changedBy) {}

File: frontend/Exence/src/app/data-model/modules/audit-log/AuditLogFilter.ts
import { AuditableEntityType } from './AuditableEntityType';
import { ChangeType } from './ChangeType';

export interface AuditLogFilter {
	entityType: AuditableEntityType;
	from: Date;
	to: Date;
	changeType: ChangeType;
	changedBy: string;
}
